### PR TITLE
Documentation corrections

### PR DESCRIPTION
**Pull request by gabegreenwood**
Several small fixes to the documentation:
- Couple of broken links on Quickstart page (need to be synced with documentation restructuring by @SiaraMist)
- Typo in troubleshooting page ("ssh" should be "SSD")
- Clarify comment in Quickstart tutorial to specify *git* repo in helm chart (n00bs like me could confuse this to mean helm repo)
- Error in one of the helm commands in Quickstart tutorial: the create repo command creates a repo called `actions-runner-controller` but in the following command it is referred to simply as `actions`, which throws a "repo not found" error. Changing the repo name to `actions` fixes this and makes the second command work correctly.

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/
 ##### Add repository
 
 ```shell
-helm repo add actions-runner-controller https://actions-runner-controller.github.io/actions-runner-controller
+helm repo add actions https://actions-runner-controller.github.io/actions-runner-controller
```
Comment (gabegreenwood) [inline docs/quickstart.md]: An alternative to this change, in order to make this command work with the next one while keeping line 56 unchanged, would be to change line 65 to:
`--wait actions-runner-controller actions-runner-controller/actions-runner-controller`

**Comment by SiaraMist** [comment]
Thank you for fixing the broken links! 🙏

**Comment by SiaraMist** [comment]
Thanks again for these updates @gabegreenwood! ✨ 

@Link-, from a docs perspective, this PR won't cause any conflicts with our current progress. 👍 So if your team does a technical review and approves, this is good to merge from docs!

File: docs/quickstart.md (hunk 1 above)
Comment (Link-) [reply docs/quickstart.md]: I don't see the need for this change, the project is still called `actions-runner-controller` and using that name for the installation makes sense.

**Review by Link- — CHANGES_REQUESTED** [review]
Thank you @gabegreenwood for your contribution. I'm requesting a small change before merging these fixes. Also, kindly update your fork with the latest changes to resolve the conflicts

File: docs/quickstart.md (hunk 1 above)
Comment (gabegreenwood) [reply docs/quickstart.md]: Looks like the issue was corrected by updating line 65 in another PR so this change isn't necessary

**Comment by gabegreenwood** [comment]
@Link- I made the change you requested, my original change was no longer necessary as the issue seems to have been corrected by changing a different line in that same file.

**Comment by mumoshu** [comment]
Fixed another conflict. I believe this is good to go now!

**Review by mumoshu — APPROVED** [review]
Thank you so much for your contribution @gabegreenwood!

**Review by TingluoHuang — APPROVED** [review]

**Review by Link- — APPROVED** [review]

**Pull request merged**